components/automate-gateway/api/nodes/manager/manager.proto

```diff
@@ -136,6 +230,21 @@ service NodeManagerService {
 			action: "infra:nodeManagers:list"
 		};
 	};
+
+	/*
+	Search nodes
+
+	Searches the available nodes for a given nodemanager id.
+
+	Example:
+	```
+	{
+		"query": {
+			"filter_map":[]
+		}	
+	}
```

Comment:
Are the empty parameters necessary? Does this do something different than not providing any parameters at all?

Reply:
Every key needs a value. This one just says "use an array", which is implied by "map", but every little bit helps.